.github/script/send-test-report.js

```diff
@@ -5,11 +5,13 @@ const axios = require('axios');
 
 var myArgs = process.argv.slice(2);
 
-axios.post(myArgs[0], {
-  'github_workflow_url': myArgs[1],
-  'browser_compatibility_tests_status': myArgs[2].toUpperCase()
-}, res => {
-  console.log(res);
-}, err => {
-  console.log(err);
-});
+if (myArgs[2].toUpperCase() !== 'SUCCESS') {
```

Comment:
So we send if it is not successful ? Does not seem to be what the title of the PR implies? :D